apps/opik-backend/src/main/java/com/comet/opik/domain/ProjectDAO.java

```diff
@@ -23,7 +24,7 @@
 @RegisterConstructorMapper(Project.class)
 @RegisterConstructorMapper(ProjectIdLastUpdated.class)
 @RegisterArgumentFactory(UUIDArgumentFactory.class)
-interface ProjectDAO {
+public interface ProjectDAO {
```

Comment:
I would avoid making it public. We can expose it via the service

Reply:
Done